roles/openshift_hosted/tasks/wait_for_pod.yml

```diff
@@ -18,6 +18,9 @@
              --config={{ openshift.common.config_base }}/master/admin.kubeconfig \
              -o jsonpath='{ .status.latestVersion }'
     register: l_openshift_hosted_wfp_latest_version
+    delay: 10
+    retries: 5
+    until: l_openshift_hosted_wfp_latest_version.rc == 0
```

Comment:
This looks unrelated to the rest of the patch. And we're not really allowed to review that bit. Can you separate it out to another pull request?

Reply:
sorry, yes i'll remote it.